hawkular-inventory-parent/hawkular-inventory-service/src/main/java/org/hawkular/inventory/handlers/ManifestUtil.java

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright 2014-2017 Red Hat, Inc. and/or its affiliates
+ * and other contributors as indicated by the @author tags.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.hawkular.inventory.handlers;
+
+import java.io.IOException;
+import java.net.URL;
+import java.util.Arrays;
+import java.util.Enumeration;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.jar.Attributes;
+import java.util.jar.Manifest;
+
+/**
+ * @author Jay Shaughnessy
+ * @author Lucas Ponce
+ */
+public class ManifestUtil {
+    private static final String IMPLEMENTATION_VENDOR_ID = "Implementation-Vendor-Id";
+    private static final String IMPLEMENTATION_VERSION = "Implementation-Version";
+    private static final String BUILT_FROM_GIT = "Built-From-Git-SHA1";
+    private static final String HAWKULAR_ALERTING = "org.hawkular.inventory";
```

Comment:
HAWKULAR_ALERTING => HAWKULAR_INVENTORY

Reply:
ops, copy paste :-) I can't hide the source.
Thx.